utils.js

```diff
@@ -170,6 +170,19 @@ function findLongOptionForShort(shortOption, options) {
   return longOptionEntry?.[0] ?? shortOption;
 }
 
+/**
+ * Check if the given option includes a default value
+ * and that option has not been set by the input args.
+ *
+ * @param {array} options - option configs entry, from parseArgs({ options })
+ * @param {object} values - option values returned in `values` by parseArgs
+ */
+function useDefaultValueOption({ 0: longOption,
```

Comment:
That first destructured argument here is pretty wild! Straight out of the use case in the calling code. The JSDoc for the arguments is a copy-paste from another routine and does not match.

I suggest the arguments here could be longOption, optionConfig, values and the destructuring could be in the calling code.